mindmeld/tokenizer.py

```diff
@@ -250,9 +252,7 @@ def tokenize(self, text, keep_special_chars=True):
         Returns:
             list: A list of normalized tokens
         """
-
-        raw_tokens = self.tokenize_raw(text)
-
```

Comment:
Do we intend to keep `tokenize_raw()` defined in this class for backwards compatibility? It is public, but it seems unlikely that people would actually invoke it directly.

Reply:
In Phase 2, we will be removing tokenize_raw(). Instead each Tokenizer class (Eg "CharacterTokenizer") will have a tokenize() method. The tokenize() method will be called by the (new) TextPreparationPipeline Class.